### PR TITLE
Payton: Initial commit.

### PR DESCRIPTION
**Pull request by alaczek**
Add a new Blockbase child called Payton.

### CODE REVIEW

File: payton/functions.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Add Editor Styles
+ */
+function newtheme_editor_styles() {
```
Comment (scruffian) [inline payton/functions.php]: This should say payton, not newtheme.

File: payton/functions.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	// Enqueue editor styles.
+	add_editor_style(
+		array(
+			'/assets/theme.css',
+		)
+	);
+}
+add_action( 'after_setup_theme', 'newtheme_editor_styles' );
```
Comment (scruffian) [inline payton/functions.php]: This should say payton, not newtheme.

File: payton/functions.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ *
+ * Enqueue scripts and styles.
+ */
+function newtheme_scripts() {
```
Comment (scruffian) [inline payton/functions.php]: This should say payton, not newtheme.

**Review by scruffian — APPROVED** [review]
Just a few instances of newtheme->payton and then we can merge this

File: payton/functions.php (hunk 2 above)
Comment (alaczek) [reply payton/functions.php]: Good catch! I think this was generated by the script and I didn't notice - should the script be inserting theme name here?

Comment (scruffian) [reply payton/functions.php]: yeah, I thought i'd fixed it but looks like it's not working, or you're using an older version

Comment (alaczek) [reply payton/functions.php]: That's entirely possible :) I'll keep an eye out for it next time I'm generating a theme.

**Pull request merged**